doc/lua_api.md

```diff
@@ -7848,13 +7850,15 @@ An `InvRef` is a reference to an inventory.
   can be fully added to the list
 * `contains_item(listname, stack, [match_meta])`: returns `true` if
   the stack of items can be fully taken from the list.
-  If `match_meta` is false, only the items' names are compared
-  (default: `false`).
-* `remove_item(listname, stack)`: take as many items as specified from the
-  list, returns the items that were actually removed (as an `ItemStack`)
-  -- note that any item metadata is ignored, so attempting to remove a specific
-  unique item this way will likely remove the wrong one -- to do that use
-  `set_stack` with an empty `ItemStack`.
+    * If `match_meta` is `true`, item metadata is also considered when comparing
+      items. Otherwise, only the items' names are compared. Default: `false`
+    * The method ignores wear.
+* `remove_item(listname, stack, [match_meta])`: take as many items as specified from the
+  list, returns the items that were actually removed (as an `ItemStack`).
+    * If `match_meta` is `true` (available since feature `remove_item_match_meta`),
+      item metadata is also considered when comparing items. Otherwise, only the
+      items' names are compared. Default: `false`
```

Comment:
i read somewhere a while ago that this is not actually correct spelling
```suggestion
      items. Otherwise, only the items names are compared. Default: `false`
    * The method ignores wear.
* `remove_item(listname, stack, [match_meta])`: take as many items as specified from the
  list, returns the items that were actually removed (as an `ItemStack`).
    * If `match_meta` is `true` (available since feature `remove_item_match_meta`),
      item metadata is also considered when comparing items. Otherwise, only the
      items names are compared. Default: `false`
```